# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mplayer/parallelism.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-03-22 10:49:30.699353312 -0400
+++ ./Makefile	2005-03-22 10:50:10.445311008 -0400
@@ -187,7 +187,7 @@
 libdha/libdha.so:
 	$(MAKE) -C libdha
 
-vidix/libvidix.a:
+vidix/libvidix.a: libdha/libdha.so
 	$(MAKE) -C vidix
 
 Gui/libgui.a:
